modules/mobile/app/controllers/mobile/v0/payment_history_controller.rb

```diff
@@ -2,7 +2,6 @@
 
 require_dependency 'mobile/application_controller'
 require 'adapters/payment_history_adapter'
-require 'mobile/v0/exceptions/validation_errors'
```

Comment:
The reason we were not getting errors when using the validations is that this file was requiring the validation errors file. Because this file is auto loaded, the validation errors were piggy-backing this file.